lib/routes/data/styles.js

```diff
@@ -63,4 +63,16 @@ module.exports = {
       ],
     ],
   },
+  alertData: [
+    {
+      for: '00eef724-7cbd-47f5-a735-630510f39711',
+      type: 'Warning!',
```

Comment:
The alert for forms does not currently have two options. It is just a single message. I think this data should just be `for` and `value`.